src/ledger/autofill.ts

```diff
@@ -20,13 +20,13 @@ interface ClassicAccountAndTag {
 /**
  * Autofills fields in a transaction.
  *
- * @param client - A client.
+ * @param this - A client.
  * @param transaction - A transaction to autofill fields.
  * @param signersCount - The expected number of signers for this transaction. Used for multisign.
  * @returns An autofilled transaction.
  */
 async function autofill<T extends Transaction>(
-  client: Client,
+  this: Client,
```

Comment:
Is it safe to name a param using `this` keyword?

Reply:
The `Client` parameter is required to be named `this` if you want to bind it to a class member.